doc/signal_path.rst

```diff
@@ -0,0 +1,302 @@
+Signal path overview
+====================
+
+This section gives a logical overview of the signal path. The actual
+implementation is mathematically equivalent (when ignoring floating-point
+rounding errors), but it splits, combines, and reorders steps for efficiency;
+refer to the :doc:`fgpu.design` and :doc:`xbgpu.design` sections for details.
+
+Edges in diagrams are annotated to indicate the data type. The following types
+are used:
+
+:samp:`i{N}`
+  Signed integer or fixed-point with :samp:`{N}` total bits
+:samp:`f{N}`
+  Floating-point with :samp:`{N}` bits, following IEEE 754-2019.
+:samp:`c{X}`
+  Complex values formed from real and imaginary components of type :samp:`{X}`
+  e.g., ``cf32`` is single-precision floating point complex.
+
+Dotted boxes and arrows represent control parameters that can be adjusted at
+runtime.
+
+Channelisation
```

Comment:
Bike-shedding: I realise you're broadly talking about the F- part of an F/X correlator, but I think it would be clearer to add something like "and delay correction" to this heading because there's a good deal more going on here than just channelisation.